services/management/ia/templates_service.go

```diff
@@ -151,15 +286,35 @@ func (svc *TemplatesService) loadFile(ctx context.Context, file string) ([]saas.
 
 	// be strict about local files
 	params := &saas.ParseParams{
-		DisallowUnknownFields: true,
-		DisallowInvalidRules:  true,
+		DisallowUnknownFields:    true,
+		DisallowInvalidTemplates: true,
 	}
-	rules, err := saas.Parse(bytes.NewReader(data), params)
+	templates, err := saas.Parse(bytes.NewReader(data), params)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to parse rule template file")
 	}
 
-	return rules, nil
+	return templates, nil
+}
+
+func convertParamType(t alert.Type) iav1beta1.ParamType {
+	// TODO: add another types.
+	switch t {
+	case alert.Float:
+		return iav1beta1.ParamType_FLOAT
+	default:
+		return iav1beta1.ParamType_PARAM_TYPE_INVALID
+	}
+}
+
+func convertParamUnit(u string) iav1beta1.ParamUnit {
+	// TODO: check possible variants.
+	switch u {
+	case "%", "percentage":
+		return iav1beta1.ParamUnit_PERCENTAGE
+	default:
+		return iav1beta1.ParamUnit_PARAM_UNIT_INVALID
+	}
 }
```

Comment:
There is no any validation for unit on alert.Template side, it should be just non-empty string. Here we have enum.